### ISSUE TITLE
IDs on Fulcrum Home Page Should Be Unique

### ISSUE THREAD

**Issue by jmcglone** [mlibrary/heliotrope#607]
Document has multiple elements with the same id attribute: button--get-updates

`#index-banner > .section.no-pad-bot > .container > div:nth-of-type(2) > .btn-large.button--lg.waves-effect.waves-light.lighten-1`

`<a href="#get-updates" id="button--get-updates" class="btn-large button--lg waves-effect waves-light lighten-1">Get Updates</a>`
